pkgs/misc/drivers/utsushi/default.nix

```diff
@@ -37,6 +37,7 @@ in stdenv.mkDerivation rec {
       url = "https://raw.githubusercontent.com/archlinux/svntogit-community/b3046e0e78b95440f135fcadb19a9eb531729a58/trunk/boost-1.74.patch";
       sha256 = "sha256-W8R1l7ZPcsfiIy1QBJvh0M8du0w1cnTg3PyAz65v4rE=";
     })
+    ./sane_version.patch
```

Comment:
```suggestion
    # fix build with sane version 1.1 and above
    ./sane_version.patch
```

Reply:
Thanks - I've noted that Gentoo have a patch so I've used that instead.